vignettes/customise_query.Rmd.src

```diff
@@ -0,0 +1,261 @@
+
+lx_point <- oc_points(lx$oc_lat, lx$oc_lng)
+
+oc_forward_df(placename = "Paris", proximity = lx_point, limit = 5)
+```
+Note that the French capital is listed before other places in the US, which are closer to the point provided.
+This illustrates how `proximity` is only one of many factors influencing the ranking of results.
+
+### Confidence
+
+`min_confidence`, an integer value between 0 and 10, indicates the precision of the returned result as defined by its geographical extent, (i.e. by the extent of the result's bounding box).
+When you specify `min_confidence`, only results with at least the requested confidence will be returned.
+I.e. in the following example, the French capital is too large to be returned.
+
+```{r confidence}
+oc_forward_df(placename = "Paris", min_confidence = 7, limit = 5)
+```
+
+Note that confidence is not used for the [ranking of results](https://opencagedata.com/api#ranking).
+It does not tell you which result is more "correct" or "relevant", nor what type of thing the result is, but rather how small a result is, geographically speaking.
+See the [API documentation](https://opencagedata.com/api#confidence) for details.
+
+## Retrieve more information from the API
+
+Besides parameters to target your search better, OpenCage offers parameters to receive more and specific information from the API.
+
+### `language`
+
+If you would like to get your results in a specific language, you can pass an [IETF BCP 47 language tag](https://en.wikipedia.org/wiki/IETF_language_tag), such as "tr" for Turkish or "pt-BR" for Brazilian Portuguese, to the `language` parameter.
+OpenCage will attempt to return results in that language.
+
+```{r language_it}
+oc_forward_df(placename = "Munich", language = "tr")
+```
+
+Alternatively you can specify the "native" tag, in which case OpenCage will attempt to return the response in the "official" language(s).
+Keep in mind, however, that some countries have more than one official language or that the official language may not be the one actually used day-to-day.
+
+```{r language_native}
+oc_forward_df(placename = "Munich", language = "native")
+```
+
+In case the `language` parameter is set to `NULL` (which is the default), the tag is not recognized, or OpenCage does not have a record in that language, the results will be returned in English.
+
+```{r language_null}
+oc_forward_df(placename = "München")
+```
+
+To find the correct language tag for your desired language, you can search for the language on the [BCP47 language subtag lookup](https://r12a.github.io/app-subtags/) for example.
+Note however, that there are some language tags in use on OpenStreetMap for example, one of OpenCage's main sources, that do not conform with the IETF BCP 47 standard.
+For example OSM uses [`zh_pinyin`](https://wiki.openstreetmap.org/w/index.php?title=Multilingual_names#China) instead of `zh-Latn-pinyin` for [Hanyu Pinyin](https://en.wikipedia.org/wiki/Pinyin).
```

Comment:
```suggestion
For example, OSM uses [`zh_pinyin`](https://wiki.openstreetmap.org/w/index.php?title=Multilingual_names#China) instead of `zh-Latn-pinyin` for [Hanyu Pinyin](https://en.wikipedia.org/wiki/Pinyin).
```